### PR TITLE
Added 404 Error Page

### PR DESCRIPTION
**Pull request by skks1212**
Closes [https://github.com/coronasafe/care_fe/issues/2352](https://github.com/coronasafe/care_fe/issues/2352)
image

### CODE REVIEW

**Comment by nihal467** [comment]
@skks1212 @gigincg  is it good for testing?

**Comment by dauntlessnomad** [comment]
@skks1212 can you make this into a single page html with css and we can put this globaly on our domain. 
this looks good

**Comment by shivankacker** [comment]
> @skks1212 can you make this into a single page html with css and we can put this globaly on our domain. this looks good

Suree! There you go @tomahawk-pilot [https://github.com/skks1212/care_misc/tree/master/error_pages]( https://github.com/skks1212/care_misc/tree/master/error_pages)

**Comment by dauntlessnomad** [comment]
Instead of the asset folder let's pull it from a cdn

**Comment by shivankacker** [comment]
> Instead of the asset folder let's pull it from a cdn

Cool! Can you put it up on the coronasafe cdn? (If we have one?)
Otherwise I will put up on another.

**Comment by nihal467** [comment]
test approved

**Comment by shivankacker** [comment]
@gigincg deploy is failing, can we clear cache?

File: src/Router/AppRouter.tsx [hunk 1]
```diff
@@ -364,6 +358,7 @@ const routes = {
   "/facility/:facilityId/patient/:patientId/consultation/:consultationId/treatment-summary":
     ({ patientId, consultationId }: any) => (
```
Comment (khavinshankar) [inline src/Router/AppRouter.tsx]:
```suggestion
    ({ facilityId, patientId, consultationId }: any) => (
```

_do destructure facilityId from the param before using_

**Review by khavinshankar — CHANGES_REQUESTED** [review]
please do work on the requested changes

**Comment by skks1212** [comment]
> please do work on the requested changes

I don't have access to the cdn, and i think we don't require a cdn for this PR @gigincg what should I do?

**Comment by shivankacker** [comment]
> please do work on the requested changes

@khavinshankar Done please check

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**